unit_tests/test_base_version.py

```diff
@@ -33,7 +33,7 @@ def test_master(self):
         scylla_repo = self.url_base + 'unstable/scylla/master/rpm/centos/2021-08-29T00:58:58Z/scylla.repo'
         linux_distro = 'centos'
         version_list = general_test(scylla_repo, linux_distro)
-        self.assertEqual(version_list, ['5.1'])
+        self.assertEqual(version_list, ['5.2'])
```

Comment:
I guess this commit must be removed because it is already part of another PR

Reply:
fixed